README.md

```diff
@@ -57,6 +57,11 @@ $app->get('/get', function (ServerRequestInterface $request, ResponseInterface $
 You can do this also in asynchronous contexts and long running processes,
 since no super-globals nor I/O are involved.
 
+It is recommended that you use a key with lots of entropy, preferably
```

Comment:
The key itself can be changed to a string that explains what should be done (very briefly)